publish-hook/src/test/java/com/redhat/lightblue/hook/publish/BasePublishHookTest.java

```diff
@@ -30,12 +33,20 @@ public BasePublishHookTest() throws Exception {
         super();
     }
 
+    @Override
+    public Set<String> getHooksToRemove() {
+        return new HashSet<>();
+    }
+    
     @Before
     public void before() throws UnknownHostException {
         cleanupMongoCollections("country", "esbEvents");
     }
 
     protected void insertCountry() throws Exception {
+        com.mongodb.DBCursor cursor=mongoServer.getConnection().getDB("testdb").getCollection("metadata").find();
+        while(cursor.hasNext())
+            System.out.println(cursor.next());
```

Comment:
I see. I would like for us to remove this block though. It assumes the use of testdb, which is the default but not a guarantee. Also, when debugging, you can place a break point in the asserts and connect to the in-memory mongo directly using `mongo localhost:27777`.